roles/sro_deploy_custom_commit/tasks/deploy.yml

```diff
@@ -0,0 +1,24 @@
+- block:
+  - name: Git checkout
+    git:
+      repo: "{{ sro_git_repo }}"
+      dest: /tmp/special-resource-operator
+      version: "{{ sro_git_ref }}"
+      
+  - name: Deploy the custom version of the special resource operator
+    command: make deploy TAG="{{ sro_image_tag }}" NAMESPACE="openshift-special-resource-operator" 
```

Comment:
this command has long logs (see [there](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/117/pull-ci-openshift-psap-ci-artifacts-master-gpu-operator-e2e/1389013453443698688/artifacts/gpu-operator-e2e/presubmit-operatorhub/artifacts/010621__special-resource-operator__deploy_from_commit/_ansible.log)),

in this case I prefer redirecting to a dedicated file, so that the main log file remains easy to read